src/MSRIOGroup.hpp

```diff
@@ -171,6 +172,11 @@ namespace geopm
             std::map<std::string, std::string> m_signal_desc_map;
             std::map<std::string, std::string> m_control_desc_map;
             std::map<std::string, int> m_signal_units_map;
+
+            /// @todo remove with combined signal refactoring
```

Comment:
Looking forward to seeing the combined signal refactoring.  The type for m_combined_signals is quite a container.